app/elements/socobo-ranking/socobo-ranking.html

```diff
@@ -542,7 +546,7 @@
          */
         _reload: function() {
           // notify user
-          this.notify("Reloading...");
+          this.notify("Ranking - Reloading...");
```

Comment:
That sounds good. I change The toasts to this behavior. Should i only show The toast if i hit the reload button or very time?